### DISCUSSION TITLE
Faker as API - fakend.fyi

### DISCUSSION THREAD

**Discussion by sanjaykhanssk** [faker-js/faker#2619] [Q&A] [answered]
Hi all, i have created this project [fakend.fyi ](https://fakend.fyi) a free mock API tool basked on faker. I need your help to validate the product. I hope you guys will share reviews about the product.

**Comment by matthewmayer** [comment]
Some friendly advice

- if you're building this on top of faker.js it would be good to credit faker.js as the source of the data on your homepage and link to the project home page. There's nothing wrong with building a commercial product on top of an open source library but you should be transparent about it..
- don't claim to be no 1 on product hunt if you are not 
- I think your testimonials look quite fake (I don't believe this is Alice 😀). I understand when you're building a new product you won't have testimonials yet. In this case, I think you'll be better to make a joke of it and say hey, "here are some fake testimonials that you could generate using this product"

![image](https://github.com/faker-js/faker/assets/152770/187a566e-3b40-4042-b50e-f036e1bfe507)

**Reply by sanjaykhanssk** [reply]
cool, thanks for the input 😃

**Comment by Shinigami92** [comment, marked as answer]
You should open an issue or PR in https://github.com/faker-js/awesome-faker
Then we will check it
